src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/CompositeMLDsa/CompositeMLDsaTestHelpers.cs

```diff
@@ -468,7 +468,7 @@ internal static int ExpectedPrivateKeySizeLowerBound(CompositeMLDsaAlgorithm alg
                 ExecuteComponentFunc(
                     algorithm,
                     rsa => rsa.KeySizeInBits / 8,
-                    ecdsa => (ecdsa.KeySizeInBits + 7) / 8,
+                    ecdsa => ((ecdsa.KeySizeInBits + 7) / 8) + 7, // Overhead for ECPrivateKey
```

Comment:
Is this making for a "tigher lower bound"?  If not, what does this change mean?

Reply:
Yes, previously the lower bound was just "ECPrivateKey needs at least the raw private key". This PR tightens the bound to the optimal lower bound by factoring in the overhead.

The main reason I changed it is because draft 8 has specified that ECPrivateKey must omit the optional fields. So the size of the key is now exact - the optimal upper bound and lower bounds are the same. So it just made sense to use that value everywhere since we have it.